### ISSUE TITLE
Pls tell me why derive the discriminator loss function w.r.t. the label rather than the input data. 

### ISSUE THREAD

**Issue by KeatsHao** [ai-tor/PyTorch-CycleGAN#38]
Pls tell me why derive the discriminator loss function w.r.t. the label rather than the input data.
